pandas/tests/categorical/test_constructors.py

```diff
@@ -468,23 +469,23 @@ def test_from_codes_with_categorical_categories(self):
     def test_from_inferred_categories(self, dtype):
         cats = ['a', 'b']
         codes = np.array([0, 0, 1, 1], dtype='i8')
-        result = Categorical._from_inferred_categories(cats, codes, dtype)
```

Comment:
would be ok with removing these tests (as I think you have them covered directly in the parser)